yarn.lock

```diff
@@ -2022,6 +2022,23 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@commercetools-test-data/product-discount@workspace:models/product-discount":
+  version: 0.0.0-use.local
+  resolution: "@commercetools-test-data/product-discount@workspace:models/product-discount"
+  dependencies:
+    "@babel/runtime": ^7.17.9
+    "@babel/runtime-corejs3": ^7.17.9
+    "@commercetools-test-data/commons": 4.6.0
+    "@commercetools-test-data/core": 4.6.0
+    "@commercetools-test-data/product-discount-value-absolute": 4.6.0
+    "@commercetools-test-data/product-discount-value-external": 4.6.0
+    "@commercetools-test-data/product-discount-value-relative": 4.6.0
```

Comment:
@emmenko What're your thoughts on these discount value models? Should they all be part of one package?

I broke them out for obviousness but was wondering whether union type models should be grouped in the same directory.